pkg/tenantNamespace/namespaceManager.go

```diff
@@ -123,6 +115,23 @@ func (nm *tenantNamespaceManager) CreateNamespace(ctx context.Context, cluster l
 	return ns, nil
 }
 
+// GenNamespaceObj returns a Tenant Namespace resource object given name and clusterid.
+func (nm *tenantNamespaceManager) GenNamespaceObj(cluster liqov1alpha1.ClusterID, name string) (ns *v1.Namespace) {
+	return &v1.Namespace{
```

Comment:
I would call this function Forge to be more coherent with the terminology used in the rest of the codebase and on liqoctl.

```suggestion
// ForgeNamespace returns a Tenant Namespace resource object given name and remote clusterID.
func (nm *tenantNamespaceManager) ForgeNamespace(clusterID liqov1alpha1.ClusterID, name string) *v1.Namespace {
```